plugins/nf-amazon/src/test/nextflow/cloud/aws/batch/AwsContainerOptionsMapperTest.groovy

```diff
@@ -94,7 +94,25 @@ class AwsContainerOptionsMapperTest extends Specification {
         def map = CmdLineHelper.parseGnuArgs('--shm-size 12048024')
         def properties = AwsContainerOptionsMapper.createContainerProperties(map)
         then:
-        properties.getLinuxParameters().getSharedMemorySize() == 12048024
+        properties.getLinuxParameters().getSharedMemorySize() == 11
```

Comment:
Because AWS measures shared memory size in MiB rather than bytes (rather than the docker standard of bytes), the `AwsContainerOptionsMapper` should do the conversion automatically.

Reply:
Well spotted Rob. Just made a minor change to make it more explicit 548d40a7